spec/requests/classified_listings_spec.rb

```diff
@@ -209,11 +255,19 @@
 
   describe "PUT /listings/:id" do
     let(:listing) { create(:classified_listing, user: user) }
+    let(:listing_draft) { create(:classified_listing, user: user) }
     let(:organization) { create(:organization) }
     let(:org_listing) { create(:classified_listing, user: user, organization: organization) }
+    let(:org_listing_draft) { create(:classified_listing, user: user, organization: organization) }
 
     before do
       sign_in user
+      listing_draft.bumped_at = nil
```

Comment:
You can use `update_columns` here for the faster tests. Probably, validations and callbacks are not needed here.